### ISSUE TITLE
De-pluralise JArray members

### ISSUE THREAD

**Issue by SteGriff** [kuhnboy/JsonXslt#1]
Hi, thanks for the project, it's fantastic!

I'm using the project to convert JSON to XML for a research-based web server. I don't know if this is applicable to everyone, but I've made a change so that when a JSON array like this

```
"slides" : [ {"title" : "Animals"}, {"title" : "Plants"} ]
```

Is converted to XML, instead of

```
<slides>
    <slides>
        <title>Animals</title>
    </slides>
    <slides>
        <title>Plants</title>
    </slides>
</slides>
```

Each member of the array has it's name de-pluralised, which is more natural to me.

```
<slides>
    <slide>
        <title>Animals</title>
    </slide>
    <slide>
        <title>Plants</title>
    </slide>
</slides>
```

Should I make a pull request or is my use case too esoteric?
Thanks again for making this thing! :smiley:

**Comment by kuhnboy** [comment]
Hi. Thanks for the feedback. This may work for simple english words, but may not work in terms of words like 'buses' would become 'buse'. I don't think I could handle all cases with this.

**Comment by SteGriff** [comment]
Yep, fair enough. EntityFramework does something like this but has a smarter algorithm with edge cases like the one you mentioned.  

What do you think about de-pluralising _in principle_? :)
